doc/nrf/libraries/security/trusted_storage.rst

```diff
@@ -84,7 +84,11 @@ Before using the trusted storage library with its default settings and options,
 * Zephyr's settings subsystem has to be enabled for use by setting the Kconfig option :kconfig:option:`CONFIG_SETTINGS`.
 
   * The settings subsystem uses the :ref:`zephyr:nvs_api` file system by default.
-    This file system has to be mounted to a mount point at application startup. For more information about this, see :ref:`zephyr:file_system_api`.
+    This file system has to be mounted to a mount point at application startup.
+    For more information about this, see :ref:`zephyr:file_system_api`.
+  * For nRF54L Series devices, you can use the :ref:`zephyr:zms_api` file system as the default settings subsystem storage.
```

Comment:
As commented in the release notes this is not only for the 54L Series devices. It's that for the 54L Series devices it's the only option permitted from now on. For other devices both are possible.